### DISCUSSION TITLE
社区简介是不是有点迷惑的地方。。。

### DISCUSSION THREAD

**Discussion by MikeLing** [gsoc-cn/gsoc-cn#120] [General]
现在的是 `Summer of Code SIG 编程之夏兴趣组（中国）`
换成 `Summer of Code && SIG 编程之夏兴趣组（中国）` 会好点🤔

**Comment by gaocegege** [comment]
没有 get，&& 在这里是“和”的意思么

**Reply by MikeLing** [reply]
嗯嗯。。。因为我觉得SoC和SIG不是一个意思（
